app/views/work_orders/cancellations/new.erb

```diff
@@ -0,0 +1,80 @@
+<% content_for :title, "Cancel Works Order: #{@work_order.reference}" %>
+<% content_for :back_button do %>
+  <%= link_to 'Back', work_order_path(@work_order.reference), class: 'govuk-back-link' %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l">
+        Cancel repair
+      </span>
+      <h1 class="govuk-heading-l">
+        Works order: <%= @work_order.reference %>
+      </h1>
+
+      <table class="govuk-table">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">
+            Property
+          </th>
+          <td class="govuk-table__cell">
+            <%= @work_order.property.description.singularize %>,
+            <%= @work_order.property.first_line_of_address %>
+          </td>
+        </tr>
+
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">
+            Trade
+          </th>
+          <td class="govuk-table__cell">
+            <%= @work_order.trade %>
+          </td>
+        </tr>
+
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">
+            Description
+          </th>
+          <td class="govuk-table__cell">
+            <%= @work_order.problem_description %>
+          </td>
+        </tr>
+      </table>
+
+      <%= govuk_form_with(
+        model: @cancellation,
+        url: work_order_cancellation_path(@work_order.reference),
+        local: true
+      ) do |form|
+      %>
+
+        <%= form.form_group :reason do %>
+          <%= form.label :reason, "Reason for cancelling" %>
+          <%= form.error_messages :reason %>
+          <%= form.text_area :reason, class: 'govuk-textarea govuk-!-margin-bottom-1', maxlength: 200, rows: 4, onkeyup: 'characterCount(event)' %>
+          <span class="govuk-hint">
+            You have <span id="counter" data-maximum-length="200">200</span> characters remaining.
+          </span>
+        <% end %>
+
+        <% if @work_order.is_dlo? && @work_order.servitor_reference.blank? %>
+          <%= render layout: "shared/govuk_warning_text" do %>
+            This works order does not have a Servitor reference. Please wait 30
+            minutes and then cancel this Works Order. If the work was due to take
+            place today, please call the Planners before cancelling the Works
+            Order.
+          <% end %>
```

Comment:
'works order' on line 64 doesn't match 'Works Order' on lines 66-67

Reply:
should be good now!